src/main/scala/org/broadinstitute/dsde/firecloud/model/Entity.scala

```diff
@@ -11,12 +11,12 @@ import scala.annotation.meta.field
 @ApiModel(value = "Entity")
 case class Entity  (
                               @(ApiModelProperty@field)(required = true, value = "The namespace the entity belongs to")
-                              wsNamespace: Option[String] = None,
+                              wsNamespace: String,
                               @(ApiModelProperty@field)(required = true, value = "The name of the workspace the entity belongs to")
-                              wsName: Option[String] = None,
+                              wsName: String,
                               @(ApiModelProperty@field)(required = true, value = "The type of the entity")
-                              entityType: Option[String] = None,
+                              entityType: String,
                               @(ApiModelProperty@field)(required = true, value = "The name of the entity")
-                              entityName: Option[String] = None,
+                              entityName: String,
                               @(ApiModelProperty@field)(required = true, value = "The attributes of the entity")
-                              attributes: Option[Map[String, String]] = None)
\ No newline at end of file
+                              attributes: Map[String, String])
```

Comment:
Using option-ed values is generally a lot easier to deal with when working with potential null values, matching blocks, etc.